environment/api_rds.tf

```diff
@@ -40,10 +41,92 @@ resource "aws_db_instance" "api" {
   }
 }
 
+resource "aws_rds_cluster" "api" {
+  count                        = local.account.always_on ? 0 : 1
+  cluster_identifier           = "api-${local.environment}"
+  engine                       = "aurora-postgresql"
+  engine_mode                  = local.account.always_on ? "provisioned" : "serverless"
+  engine_version               = "10.7"
+  availability_zones           = ["eu-west-1a", "eu-west-1b", "eu-west-1c"]
+  database_name                = "api"
+  master_username              = "digidepsmaster"
+  master_password              = data.aws_secretsmanager_secret_version.database_password.secret_string
+  skip_final_snapshot          = true
+  backup_retention_period      = 14
+  preferred_backup_window      = "07:00-09:00"
+  db_subnet_group_name         = local.account.db_subnet_group
+  kms_key_id                   = data.aws_kms_key.rds.arn
+  storage_encrypted            = true
+  vpc_security_group_ids       = [module.api_rds_security_group.id]
+  deletion_protection          = local.account.always_on ? true : false
+  enable_http_endpoint         = local.account.always_on ? false : true
+  preferred_maintenance_window = "sun:01:00-sun:01:30"
+
+  tags = merge(
+    local.default_tags,
+    {
+      "Name" = "api.${local.environment}.${local.account.account_id}.${data.aws_route53_zone.public.name}"
+    },
+  )
+
+  lifecycle {
+    ignore_changes = [engine_version, master_password]
+  }
+}
+
+# resource "aws_rds_cluster_instance" "api" {
```

Comment:
Is this commented out bit supposed to be there?

Reply:
Kind of. We'll need it when we switch our always on databases to provisioned Aurora and I didn't want to lose it.